### PR TITLE
Closes #3671

### LINKED ISSUE #3671: ui.lantern.io cannot be reached
This one was reported by @Rice321 and @Derekf5 on current 2.1.0 QA.

Looks like the browser is trying to do DNS resolution instead of letting Lantern handle it.

### PR DESCRIPTION
**Pull request by oxtoacart**
cc: @myleshorton

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -186,7 +186,6 @@ $(RESOURCES_DOT_GO): $(NPM)
 	git update-index --assume-unchanged $$APP/js/revision.js && \
 	DEST="$@" && \
 	cd $$LANTERN_UI && \
-	rm -rf node_modules && \
```
Comment (oxtoacart) [inline Makefile]: This was a bonus.  I noticed that it was left over from my Makefile debugging.

File: src/github.com/getlantern/edgedetect/edgedetect_windows.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	defer k.Close()
+
+	s, _, err := k.GetStringValue("AppUserModelId")
+	if err != nil {
+		log.Tracef("Error reading AppUserModelId: %v", err)
+		return false
+	}
+
+	log.Tracef("AppUserModelId: %v", s)
+	return strings.Contains(s, "MicrosoftEdge")
```
Comment (oxtoacart) [inline src/github.com/getlantern/edgedetect/edgedetect_windows.go]: I manually tested this using the cmd.exe program and changing the default browser from/to Edge.

**Comment by oxtoacart** [comment]
BTW, @xiam, can you please remove the dns entry for ui.lantern.io?

**Comment by myleshorton** [comment]
LGTM @oxtoacart thanks.

**Comment by myleshorton** [comment]
I just went ahead and removed that dns entry @oxtoacart

**Comment by oxtoacart** [comment]
@myleshorton Thanks!

**Comment by myleshorton** [comment]
No problem thank you!

**Pull request merged**